bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -292,7 +292,7 @@
 		<category>BatteryLevel</category>
 		<state pattern="%.0f %unit%" readOnly="true"/>
 	</channel-type>
-	<channel-type id="loadpointVehicleTitle">
+	<channel-type id="vehicleTitle">
```

Comment:
Now we have duplicate channel type …
Just remove this one with id `vehicleTitle` here, so we only have the one below.

Probably you could add a comment to the one below like `Channel Types shared across loadpoints and vehicles` so it is clear it is used in both channel groups.

Reply:
From my perspective of view it's good to know which vehicle is connected to the loadpoint I'm looking at, but also the title from every vehicle.
If I have to vehicles and one loadpoint, it's useful to have a channel for each vehicle to get their titles, but I also want to get the title of that one vehicle currently connected to the loadpoint. That's why I implemented it like this🤔

Reply:
I understand your use case, and those two different channels will be kept.

Let me explain:

Every channel has a ChannelType, and that ChannelType defines the data type (e.g. String), name and label shown in the UI, category (determines the icon shown in the UI), whether the channel is read only or not and much more.
openHAB also includes a few system-wide channel types, so for example for outdoor-temperature bindings don‘t have to define their own channel type (but they still can).
Also see https://www.openhab.org/docs/developer/bindings/thing-xml.html#channels.

This means, that sharing a channel type for the `vehicleTitle` channel of the loadpoint groups and the vehicle groups will keep those different channels, these channels will just have the same type, the same name and description etc.

Reply:
> Oh I think I understand the thing-types now..
> It just defines the channel behaviour right? So when I cut one of those two code blocks out I still got the feature I wanted to achieve?

Correct, it defines the behaviour and „look and feel“ of the channel 👍